File: totosuki/ABC/A/044.py
N = int(input())
K = int(input())
X = int(input())
Y = int(input())
rslt = 0
if N - K > 0:
  rslt = X * K
  rslt += Y * (N - K)
else:
  rslt = X * N
print(rslt)

File: OCH_Tao/ABC/A/044.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n")).map(str => parseInt(str, 10));
if (input[0] - input[1] <= 0) {
  console.log(input[0] * input[2]);
} else {
  console.log(input[1] * input[2] + (input[0] - input[1]) * input[3]);
}